aspnet/migration/rc2-to-rtm.rst

```diff
@@ -14,6 +14,16 @@ This migration guide covers migrating an ASP.NET Core RC2 application to ASP.NET
 
 There weren't many significant changes to ASP.NET Core between the RC2 and 1.0 releases. For a complete list of changes, see the `ASP.NET Core 1.0 announcements <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0>`_.
 
+Install the new tools from http://www.dot.net and follow the instructions.
```

Comment:
you need two newlines and two spaces in the code

Install the new tools from http://www.dot.net and follow the instructions.  
Update the *global.json* to

.. code-block:: javascript 
     
  {  
    "projects": [ "src", "test" ],  
    "sdk": {  
  	"version": "1.0.0-preview2-003121"  
    }  
  }